saleor/graphql/checkout/mutations/checkout_create.py

```diff
@@ -180,6 +182,26 @@ class CheckoutCreateInput(BaseInputObjectType):
         description=("The checkout validation rules that can be changed."),
     )
 
+    metadata = NonNullList(
+        MetadataInput,
+        description=(
+            f"Checkout public metadata. {ADDED_IN_321}"
+            f"{MetadataInputDescription.PUBLIC_METADATA_INPUT}"
+        ),
+        required=False,
+    )
+
+    private_metadata = NonNullList(
+        MetadataInput,
+        description=(
+            "Checkout private metadata. Requires one of the following permissions:"
+            f"{CheckoutPermissions.MANAGE_CHECKOUTS.name}, "
+            f"{CheckoutPermissions.HANDLE_CHECKOUTS.name} {ADDED_IN_321}"
```

Comment:
```suggestion
            f"{CheckoutPermissions.HANDLE_CHECKOUTS.name} {ADDED_IN_321} "
```
See the schema. There is missing space between ADDED_IN and MetadataInputDescription

Reply:
done